### ISSUE TITLE
Abort loading of torrent couses pulsar to crash

### ISSUE THREAD

**Issue by prilly** [steeve/plugin.video.pulsar#93]
if you browse any movie list with pulsar and accidently starts a movie the process popup get loaded with an abort button, if you use this button early (when scrapers is running, before torrents is handed of to pulsar and pulsar starts loading it up, pulsar will crash requiring a full reboot, this is a reproducable bug.

**Comment by NoGood** [comment]
Yeah this has happened to me too. Now I try to avoid it so I dont know if it always happens. 
You should paste a log.
